image_client.py

```diff
@@ -154,6 +156,43 @@ def check_image_db_if_filename_imported(self, collection, filename, exact=False)
 
         return self.decode_response(params)
 
```

Comment:
Function for updating exif image data on import, uses new api method for server.